lib/src/screens/task_list.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+class TaskListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: Key('task_list_screen'),
+      appBar: AppBar(title: Text('Task List')),
+      body: BodyLayout(),
+    );
+  }
+}
+
+class BodyLayout extends StatelessWidget {
+  List<String> titles = [
+    'Finish Launch Pad Issue 2',
+    'Write blog post for team website',
+    'Clean out closet',
+    'In-class exercise CPSC 304',
+    'Midterm essay ASIA 311'
+  ];
+  List<String> subtitles = [
+    'Launch Pad - 215 minutes',
+    'Triathlon - 47 minutes',
+    'Apartment - 74 minutes',
+    'Assignments - 23 minutes',
+    'Assignments - 201 minutes'
+  ];
+  @override
+  Widget build(BuildContext context) {
+    return _myListView(context);
+  }
+
+  Widget _myListView(BuildContext context) {
```

Comment:
Bit of a nitpick, but a more descriptive name than `_myListView` would make it a bit easier/faster to read and understand the code.